Upstream: Yes (fixed in git version)
Reason: Add missing include to fix building with latest kernels
Source: Dirk Mallunat <s4msung@exigen.org>

--- a/utils/dvb/dvb_frontend.h	2012-08-03 18:03:44.831131089 +0200
+++ b/utils/dvb/dvb_frontend.h	2012-08-03 18:03:33.631844968 +0200
@@ -26,6 +26,7 @@
 #ifndef _DVBFRONTEND_H_
 #define _DVBFRONTEND_H_
 
+#include <stddef.h>
 #include <linux/types.h>
 
 typedef enum fe_type {
